### PR TITLE
change the default order from asc to desc

### PR DESCRIPTION
**Pull request by zyjiaobj**

### CODE REVIEW

File: src/cli/commands/retrieve_metrics.go [hunk 1]
```diff
@@ -15,7 +15,7 @@ type MetricsCommand struct {
 	RequiredlArgs MetricsPositionalArgs `positional-args:"yes"`
 	StartTime     string                `long:"start" description:"start time of metrics collected with format \"yyyy-MM-ddTHH:mm:ss+/-HH:mm\" or \"yyyy-MM-ddTHH:mm:ssZ\", default to very beginning if not specified."`
 	EndTime       string                `long:"end" description:"end time of the metrics collected with format \"yyyy-MM-ddTHH:mm:ss+/-HH:mm\" or \"yyyy-MM-ddTHH:mm:ssZ\", default to current time if not speficied."`
-	Desc          bool                  `long:"desc" description:"display in descending order, default to ascending order if not specified."`
+	Asc           bool                  `long:"asc" description:"display in ascending order, default to descending order if not specified."`
```
Comment (cdlliuy) [inline src/cli/commands/retrieve_metrics.go]: Some users are already using "--desc" flag in their scripts.  Then,  with this change,  the original script will fail with 
```
>> cf asm APPNAME memoryused --desc
FAILED

Error: unknown flag `desc'
```
How about keeping the "Desc" flag definition here to make the above cmd are still work ?  
For Metadata and documents,  we can replace "desc"    to "asc". 

Anyway, the cmd "cf asm APPNAME memoryused --desc --asc"  will be acceptable as well according to above suggestion , but only "--asc" take effect here.

Comment (zyjiaobj) [reply src/cli/commands/retrieve_metrics.go]: that's better

Comment (zyjiaobj) [reply src/cli/commands/retrieve_metrics.go]: @boyang9527 fixed in the commit

File: src/cli/ui/message.go [hunk 2]
```diff
@@ -34,6 +34,7 @@ const (
 	UnrecognizedTimeFormat = "Unrecognized date time format: %s. \nSupported formats are yyyy-MM-ddTHH:mm:ss+/-hhmm, yyyy-MM-ddTHH:mm:ssZ with an input later than 1970-01-01T00:00:00Z."
 	UnrecognizedMetricName = "Unrecognized metric name: %s. \nSupported value: memoryused, memoryutil, responsetime, throughput, cpu."
 	InvalidTimeRange       = "Invalid time range. The start time %s is greater than the end time %s."
+	ConflictDisplayOrder   = "Conflict display order. Descending and ascending can't be used at the same time."
```
Comment (cdlliuy) [inline src/cli/ui/message.go]: DeprecatedDescWarning = "The default order is set to descending now.  Please remove the DEPRECATED flag \"--desc\" ".

File: src/cli/commands/retrieve_history.go [hunk 3]
```diff
@@ -33,6 +33,9 @@ func (command HistoryCommand) Execute([]string) error {
 		err    error
 		writer *os.File
 	)
+	if command.Desc && command.Asc {
+		return fmt.Errorf(ui.ConflictDisplayOrder)
```
Comment (cdlliuy) [inline src/cli/commands/retrieve_history.go]: How about adding a "deprecation" notice when --desc is detected?

Comment (zyjiaobj) [reply src/cli/commands/retrieve_history.go]: then:
(1) for `cf asm xxx xxx --desc --asc`, return `Failed` and show the "deprecation" notice
(2) for `cf asm xxx xxx --desc`, return results and show the "deprecation" notice
is that right?

Comment (cdlliuy) [reply src/cli/commands/retrieve_history.go]: yes, exactly.

**Comment by cdlliuy** [comment]
LGTM .

**Pull request merged**